include/sst/waveshapers/WaveshaperConfiguration.h

```diff
@@ -30,6 +30,8 @@ enum class WaveshaperType
     wst_singlefold,
     wst_dualfold,
     wst_westfold,
+    wst_linearfold,
+    wst_sinefold,
```

Comment:
Annoyingly, in surge and rack right now we stream wavetable ids by enum value.

So you will need to add your two new additions at the end of the num list not the middle.